site/content/docs/5.2/components/buttons.md

```diff
@@ -6,7 +6,19 @@ group: components
 toc: true
 ---
 
-## Examples
```

Comment:
IMO we should keep a header between this part (called "About" or "Overview" or something else) in order to be consistent with our other components; we never have a component whose doc begins without any h1 or h2.

Reply:
ah, good point. i'll have a think what would work best there...

Reply:
went for "Base class" as an initial heading